module3/projects/rancid_tomatillos/evaluation.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Rancid Tomatillos Evaluation
+module: 3
+tags: react,  testing, javascript, api, cypress
+---
+
+_[Back to Rancid Tomatillos Home](./index)_
+
+## Video Submission
+
+For the project evaluation, each project team should prepare a **10 minute** video presentation. Presentations longer than 10 minutes will not be reviewed and sent back for revision. You can use whatever tool you'd like, but we recommend [Loom](https://www.loom.com/) as a solid, free option. 
```

Comment:
Something we've always mentioned in kickoffs is that videos do not need to and should not be fancy. No editing is really necessary at all. They can go into a zoom and press record. Just to prevent folks from spinning their wheels on video editing instead of code. Maybe it's not necessary to mention by this point, but it can't hurt?